src/promptflow-evals/promptflow/evals/evaluators/content_safety/flow/evaluate_with_rai_service.py

```diff
@@ -64,12 +64,18 @@ def fetch_result(operation_id: str, rai_svc_url: str, credential: TokenCredentia
         time.sleep(sleep_time)
 
 def parse_response(batch_response: List[dict], metric_name: str) -> List[List[dict]]:
+    # Fix the metric name if it's "hate_fairness"
+    # Eventually we will remove this fix once the RAI service is updated
+    key = metric_name
+    if key == "hate_fairness":
+        key = "hate_unfairness"
+
     result = {
-        metric_name: np.nan,
-        metric_name + "_score": np.nan,
-        metric_name + "_reasoning": ""
```

Comment:
reasoning --> reason ?

Reply:
This is inherited from existing eval flow. In RAI service response, it's also called `reasoning`, but I also like to call it `reason` more. Let me check with Minsoo on this.

Reply:
Minsoo agrees on this. This change is already being worked on.